build/Dockerfile

```diff
@@ -109,9 +109,20 @@ RUN make build-daemon
 #├── windows_exporter.exe
 #└── windows-instance-config-daemon.exe
 
+FROM registry.redhat.io/openshift4/ose-cli:latest AS cli
+
 FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 LABEL stage=operator
 
+# install tools required for must-gather collection script
+RUN INSTALL_PKGS=" \
+      rsync \
+      tar \
+      " && \
+    microdnf install -y $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    microdnf clean all
```

Comment:
I'm not sure I'm on board with putting this into the WMCO image.
The WMCO image is already bloated, and adding more dependencies should be justified.

Why can't this be a separate image?

Reply:
I agree with @sebsoto. In fact the enhancement calls for shipping a separate image.